src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -504,6 +504,36 @@ uct_rc_mlx5_common_post_send(uct_rc_mlx5_iface_common_t *iface, int qp_type,
     uct_rc_txqp_posted(txqp, &iface->super, res_count, fm_ce_se & MLX5_WQE_CTRL_CQ_UPDATE);
 }
 
+static UCS_F_ALWAYS_INLINE void uct_rc_mlx5_txqp_inline_iov_post(
+        uct_rc_mlx5_iface_common_t *iface, uct_rc_txqp_t *txqp,
+        uct_ib_mlx5_txwq_t *txwq, const uct_iov_t *iov, size_t iovcnt,
+        uint8_t am_id)
```

Comment:
are you going to add `AV` related params later?

Reply:
yes, along with dc_mlx implementation.